lisa/analysis/tasks.py

```diff
@@ -524,13 +525,13 @@ def f(state):
         if cpu is not None:
             df = df[df['cpu'] == cpu]
 
-        curr_state = df.curr_state
-        active_series = curr_state[
+        curr_state = df['curr_state']
+        df['active'] = curr_state[
```

Comment:
The change itself looks fine, but I wonder why we only look at `TASK_ACTIVE` and `TASK_INTERRUPTIBLE` ? Why not do `TASK_ACTIVE` means task is active, all other states means not active ("sleeping")?

Reply:
I'm not quite sure, if you positively say so then we can make this change as well.